2-multitenant/envs/non-production/README.md

```diff
@@ -0,0 +1,20 @@
+<!-- BEGINNING OF PRE-COMMIT-TERRAFORM DOCS HOOK -->
+## Inputs
+
+| Name | Description | Type | Default | Required |
+|------|-------------|------|---------|:--------:|
+| envs | Environments | <pre>map(object({<br>    billing_account    = string<br>    folder_id          = string<br>    network_project_id = string<br>    network_self_link  = string<br>    org_id             = string<br>    subnets_self_links = list(string)<br>  }))</pre> | n/a | yes |
+
+## Outputs
+
+| Name | Description |
+|------|-------------|
+| cluster\_membership\_ids | GKE cluster membership IDs |
+| cluster\_project\_id | Cluster Project ID |
+| cluster\_regions | Regions with clusters |
```

Comment:
let's have fleet_project_id an output (even if its the same value as cluster_project_id)

Reply:
Sure re-adding along with relevant tests